unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/MainChatNotificationsComponentView.cs

```diff
@@ -0,0 +1,258 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DCL;
+using DCL.Interface;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using System.Threading;
+using Cysharp.Threading.Tasks;
+using DG.Tweening;
+
+public class MainChatNotificationsComponentView : BaseComponentView
```

Comment:
As I commented above, lets create an `IMainChatNotificationsComponentView` interface and make `MainChatNotificationsComponentView` implement it.